app/view_models/base_view_model.rb

```diff
@@ -34,7 +34,7 @@ def params(attributes)
     end
 
     def process(&block)
-      result = rows.map(&block)
+      result = rows&.map(&block)
```

Comment:
to test whether needed or not

Reply:
It is, probably due to how we set `@rows` _slightly_ different to the others.